### PR TITLE
Task 12.5

### PR DESCRIPTION
**Pull request by AdamPawlinski**
Finish task 12.5. Ask for acceptance.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+function getQuote() {
+    $.getJSON(prefix + quoteUrl, createTweet);
+    $.ajaxSetup({ cache: false });
+}
+
+function createTweet(input) {
+    var data = input[0];
```
Comment (matwjablonski) [inline js/script.js]: you are taking first element from array input. from where you know that input is an array? check that input contain at least one element before you assign it to variable data

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by AdamPawlinski** [comment]
I added the Array.isArray method to check if input is really an array. Is that okey?

File: js/script.js [hunk 2]
```diff
@@ -7,22 +7,26 @@ function getQuote() {
 }
 
 function createTweet(input) {
-    var data = input[0];
+    if (Array.isArray(input) === true) {
```
Comment (matwjablonski) [inline js/script.js]: you can write it using negation operator :) will be shorten

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by AdamPawlinski** [comment]
Is that okey?

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**